test/modules/aws-ec2-integration.ts

```diff
@@ -17,6 +17,9 @@ import {
   runUninstall,
 } from '../helpers';
 
+const { generateKeyPairSync } = require('crypto');
+const sshpk = require('sshpk');
```

Comment:
If the `crypto` and `sshpk` support `import` statements rather than `require`, I'd prefer the normal `import` ones here.